docs/tutorials/htcondor.md

```diff
@@ -0,0 +1,189 @@
+easy for customers to deploy HPC environments on Google Cloud.
+
+This tutorial will walk you through deploying a simple HTCondor pool on Google
+Cloud using the HPC Toolkit.
+
+## Select a Project
+
+Select a project in which to deploy an HPC cluster on Google.
+
+<walkthrough-project-setup billing="true"></walkthrough-project-setup>
+
+Once you have selected a project, click START.
+
+## Add Credits to the Project
+
+Talk with your tutorial leader to see if Google Cloud credits are available.
+
+## Enable APIs & Permissions
+
+In a new Google Cloud project there are several apis that must be enabled to
+deploy your HPC cluster. These will be caught when you perform `terraform apply`
+but you can save time by enabling them now by running:
+
+<walkthrough-enable-apis apis="storage.googleapis.com,compute.googleapis.com,secretmanager.googleapis.com"></walkthrough-enable-apis>
+
+## Build the Toolkit Binary
+
+To build HPC Toolkit binary from source run:
+
+```bash
+make
+```
+
+You should now have a binary named ghpc in the current directory. To verify the
+build run:
+
+```bash
+./ghpc --version
+```
+
+This should show you the version of the HPC Toolkit you are using.
+
+(Optional) To install the `ghpc` binary in your home directory under bin,
+run the following command:
+
+```bash
+make install
+exec $SHELL -l
+```
+
+## Generate a Deployment
+
+To create a deployment, an input blueprint file needs to be written or adapted
+from one of the examples found in the `examples/` or `community/examples`
+directories.
+
+This tutorial will use community/examples/htcondor-pool.yaml, which provisions
+a basic auto-scaling HTCondor pool.
+
+* a new VPC network secured from the public internet
+* an HTCondor Access Point for users to submit jobs
+* an HTCondor Central Manager that will operate the pool
+* a Managed Instance Group to scale a pool of HTCondor Execute Points to serve
+  new jobs as they are submitted
+
+The blueprint community/examples/htcondor-pool.yaml should be open in the Cloud
+Shell Editor (on the left).
+
+This file describes the cluster you will deploy. After you have inspected the
+file, use the ghpc binary to create a deployment directory by running:
+
+```bash
+./ghpc create community/examples/htcondor-pool.yaml --vars "project_id=<walkthrough-project-id/>"
+```
+
+> **_NOTE:_** The `--vars` argument is used to override `project_id` in the
+> blueprint variables. The `--vars` argument supports comma-separated list of
+> name=value variables to override blueprint variables. This feature only
+> supports variables of string type.
+
+This will create a deployment directory named `htcondor-001/`, which
+contains the terraform needed to deploy your cluster.
+
+## Deploy the Cluster
+
+Use the following commands to run terraform and deploy your cluster.
+
+```bash
+terraform -chdir=htcondor-001/htcondor init
+terraform -chdir=htcondor-001/htcondor validate
+terraform -chdir=htcondor-001/htcondor apply -auto-approve
```

Comment:
While the transient `terraform` failures are out of our control, we should add a note for users explaining that they can re-run the command if they experience those failures. Otherwise, users may be unable to find a "fix" and give up entirely.

Reply:
I've put in some text advising users to apply again. I don't really like advising it but the Cloud Shell environment has recently had IPV6 issues that cause transient failures during API interactions that are outside of our ability to resolve.